guides/ssb-server-context.md

```diff
@@ -0,0 +1,36 @@
+# ssb-server, ssb-client & Core Functions
+
+## What is ssb-server?
+[ssb-server](https://github.com/ssbc/ssb-server) is the component of secure-scuttlebutt that performs the peer-to-peer connections, and reading and writing to/from the append-only log. It can also be extended with plugins. It exposes via its API core functionality such as reading from, and writing to, the append-only log, as well as the extended functions provided by plugins utilized. This process should be thought of as a sort of "background" process, since it can only perform the peer-to-peer "gossiping" functions if it is running. An ssb-server is one to one coupled with an "identity" which is essentially the secret key associated with a given append-only log.
```

Comment:
there's some redundancy in here. Please just say the stuff about plugins + read/write once <3

Change it to say "an ssb-server is _currently_ one-to-one .."  .. .this might change in the future